src/System.Device.Gpio/System/Device/Spi/DataFlow.cs

```diff
@@ -0,0 +1,15 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Device.Spi
+{
+    /// <summary>
+    /// Defines order in which data is send over the wire.
+    /// </summary>
+    public enum DataFlow
```

Comment:
I left a comment on the issue, but I'll leave here just in case.

Found a [reference ](https://python-periphery.readthedocs.io/en/latest/spi.html)where a Python implementation names it BitOrder.  Not sure I'm set on any of these so far.  Just throwing it out for discussion.